# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1744574139 -7200
# Parent  6fd15a4ba6337a021b79cf632edf867a7dc11c89
Feature: Set icon on more stand-alone Calendar windows.

These weren't deemed necessary and removed in Bug 1833083:
https://hg-edge.mozilla.org/comm-central/rev/2c7d2f41b8ec3d8d8395b801d110a16ac902c8e0

Guess what! They are now necessary.

diff --git a/calendar/base/content/dialogs/calendar-conflicts-dialog.xhtml b/calendar/base/content/dialogs/calendar-conflicts-dialog.xhtml
--- a/calendar/base/content/dialogs/calendar-conflicts-dialog.xhtml
+++ b/calendar/base/content/dialogs/calendar-conflicts-dialog.xhtml
@@ -8,8 +8,9 @@
   id="calendar-conflicts-dialog"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   windowtype="Calendar:Conflicts"
   persist="screenX screenY"
   lightweightthemes="true"
   scrolling="false"
diff --git a/calendar/base/content/dialogs/calendar-error-prompt.xhtml b/calendar/base/content/dialogs/calendar-error-prompt.xhtml
--- a/calendar/base/content/dialogs/calendar-error-prompt.xhtml
+++ b/calendar/base/content/dialogs/calendar-error-prompt.xhtml
@@ -10,8 +10,9 @@
   id="calendar-error-prompt"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   lightweightthemes="true"
   scrolling="false"
 >
   <head>
diff --git a/calendar/base/content/dialogs/calendar-event-dialog-attendees.xhtml b/calendar/base/content/dialogs/calendar-event-dialog-attendees.xhtml
--- a/calendar/base/content/dialogs/calendar-event-dialog-attendees.xhtml
+++ b/calendar/base/content/dialogs/calendar-event-dialog-attendees.xhtml
@@ -10,8 +10,9 @@
   id="calendar-event-dialog-attendees-v2"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   windowtype="Calendar:EventDialog:Attendees"
   orient="vertical"
   persist="screenX screenY"
   lightweightthemes="true"
diff --git a/calendar/base/content/dialogs/calendar-event-dialog-recurrence.xhtml b/calendar/base/content/dialogs/calendar-event-dialog-recurrence.xhtml
--- a/calendar/base/content/dialogs/calendar-event-dialog-recurrence.xhtml
+++ b/calendar/base/content/dialogs/calendar-event-dialog-recurrence.xhtml
@@ -9,8 +9,9 @@
   id="calendar-event-dialog-recurrence"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   windowtype="Calendar:EventDialog:Recurrence"
   persist="screenX screenY width height"
   lightweightthemes="true"
   scrolling="false"
diff --git a/calendar/base/content/dialogs/calendar-event-dialog-reminder.xhtml b/calendar/base/content/dialogs/calendar-event-dialog-reminder.xhtml
--- a/calendar/base/content/dialogs/calendar-event-dialog-reminder.xhtml
+++ b/calendar/base/content/dialogs/calendar-event-dialog-reminder.xhtml
@@ -7,8 +7,9 @@
 <html
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   id="calendar-event-dialog-reminder"
   persist="screenX screenY"
   lightweightthemes="true"
   scrolling="false"
diff --git a/calendar/base/content/dialogs/calendar-event-dialog-timezone.xhtml b/calendar/base/content/dialogs/calendar-event-dialog-timezone.xhtml
--- a/calendar/base/content/dialogs/calendar-event-dialog-timezone.xhtml
+++ b/calendar/base/content/dialogs/calendar-event-dialog-timezone.xhtml
@@ -12,8 +12,9 @@
   id="calendar-event-dialog-timezone"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   windowtype="Calendar:EventDialog:Timezone"
   lightweightthemes="true"
   scrolling="false"
 >
diff --git a/calendar/base/content/dialogs/calendar-itip-identity-dialog.xhtml b/calendar/base/content/dialogs/calendar-itip-identity-dialog.xhtml
--- a/calendar/base/content/dialogs/calendar-itip-identity-dialog.xhtml
+++ b/calendar/base/content/dialogs/calendar-itip-identity-dialog.xhtml
@@ -8,8 +8,9 @@
 <html
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:html="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
+  icon="calendar-general-dialog"
   lightweightthemes="true"
   style="max-width: 400px"
 >
   <head>
diff --git a/calendar/base/content/dialogs/calendar-migration-dialog.xhtml b/calendar/base/content/dialogs/calendar-migration-dialog.xhtml
--- a/calendar/base/content/dialogs/calendar-migration-dialog.xhtml
+++ b/calendar/base/content/dialogs/calendar-migration-dialog.xhtml
@@ -11,8 +11,9 @@
   id="migration-wizard"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   branded="true"
   windowtype="Calendar:MigrationWizard"
   scrolling="false"
 >
diff --git a/calendar/base/content/dialogs/calendar-occurrence-prompt.xhtml b/calendar/base/content/dialogs/calendar-occurrence-prompt.xhtml
--- a/calendar/base/content/dialogs/calendar-occurrence-prompt.xhtml
+++ b/calendar/base/content/dialogs/calendar-occurrence-prompt.xhtml
@@ -8,8 +8,9 @@
   id="calendar-occurrence-prompt"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   lightweightthemes="true"
   scrolling="false"
 >
   <head>
diff --git a/calendar/base/content/dialogs/calendar-providerUninstall-dialog.xhtml b/calendar/base/content/dialogs/calendar-providerUninstall-dialog.xhtml
--- a/calendar/base/content/dialogs/calendar-providerUninstall-dialog.xhtml
+++ b/calendar/base/content/dialogs/calendar-providerUninstall-dialog.xhtml
@@ -8,8 +8,9 @@
   id="calendar-provider-uninstall-dialog"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   windowtype="Calendar:ProviderUninstall"
   width="480"
   height="320"
   style="min-width: 480px"
diff --git a/calendar/base/content/dialogs/calendar-uri-redirect-dialog.xhtml b/calendar/base/content/dialogs/calendar-uri-redirect-dialog.xhtml
--- a/calendar/base/content/dialogs/calendar-uri-redirect-dialog.xhtml
+++ b/calendar/base/content/dialogs/calendar-uri-redirect-dialog.xhtml
@@ -9,8 +9,9 @@
   xmlns="http://www.w3.org/1999/xhtml"
   lightweightthemes="true"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   width="600"
   scrolling="false"
 >
   <head>
diff --git a/calendar/base/content/dialogs/chooseCalendarDialog.xhtml b/calendar/base/content/dialogs/chooseCalendarDialog.xhtml
--- a/calendar/base/content/dialogs/chooseCalendarDialog.xhtml
+++ b/calendar/base/content/dialogs/chooseCalendarDialog.xhtml
@@ -7,8 +7,9 @@
 <html
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   scrolling="false"
   lightweightthemes="true"
   persist="screenX screenY"
 >
diff --git a/calendar/base/content/dialogs/publishDialog.xhtml b/calendar/base/content/dialogs/publishDialog.xhtml
--- a/calendar/base/content/dialogs/publishDialog.xhtml
+++ b/calendar/base/content/dialogs/publishDialog.xhtml
@@ -10,8 +10,9 @@
   id="calendar-publishwindow"
   xmlns="http://www.w3.org/1999/xhtml"
   xmlns:xul="http://www.mozilla.org/keymaster/gatekeeper/there.is.only.xul"
   xmlns:html="http://www.w3.org/1999/xhtml"
+  icon="calendar-general-dialog"
   windowtype="Calendar:PublishDialog"
   persist="screenX screenY"
   lightweightthemes="true"
   scrolling="false"
